modules/SofaGeneralSimpleFem/src/SofaGeneralSimpleFem/BeamFEMForceField.inl

```diff
@@ -100,7 +100,6 @@ void BeamFEMForceField<DataTypes>::bwdInit()
 {
     core::behavior::BaseMechanicalState* state = this->getContext()->getMechanicalState();
     assert(state);
-    m_matS.resize(state->getMatrixSize(),state->getMatrixSize());
```

Comment:
you are sure this can be removed without changing the behavior?

Reply:
@hugtalbot told me yes :p